src/taipy/core/_entity/_reload.py

```diff
@@ -71,7 +72,9 @@ def _do_set_entity(self, *args, **kwargs):
             ]
 
             if not self._is_in_context:
-                entity_manager._set(self)
+                entity = entity_manager._get(self, self)
```

Comment:
the purpose of this snippet is to solve the scenario where we have 2 Tasks t1 and t2:
```
assert task_1.function == print
assert task_2.function == print

task_1.function = sum
assert task_1.function == sum
assert task_2.function == sum

task_2.function = mock_func
assert task_1.function == mock_func
assert task_2.function == mock_func

assert not task_1.skippable
assert not task_2.skippable

task_1.skippable = True
assert task_1.skippable
assert task_2.skippable

task_2.skippable = False
assert not task_1.skippable
assert not task_2.skippable

assert task_1.function == mock_func
assert task_2.function == mock_func
```
Why does this matter:
Let say the initial value of `function` for both task_1 and task_2 is print
When setting task_1.function = sum => assert task_1._function == sum is True
But task_2._function == print is still True
And if we set task_2.skippable = False => auto_setter will auto set task_2 where task_2._function == print
==>> the Task entity in the repository will have its function go from print -> sum -> print